packages/datadog-instrumentations/src/azure-service-bus.js

```diff
@@ -8,57 +8,63 @@ const shimmer = require('../../datadog-shimmer')
 const dc = require('dc-polyfill')
 
 const producerCh = dc.tracingChannel('apm:azure-service-bus:send')
-
-addHook({ name: '@azure/service-bus', versions: ['>=7.9.2'], patchDefault: false }, (obj) => {
+const isItDefault = new WeakSet()
+addHook({ name: '@azure/service-bus', versions: ['>=7.9.2'] }, (obj) => {
```

Comment:
Re-add the blank line from before

```suggestion

addHook({ name: '@azure/service-bus', versions: ['>=7.9.2'] }, (obj) => {
```